Add ability to INCLUDE specific cameras instead of EXCLUDE

WietseWind opened the pull request:
Thanks for your awesome work!

Your code allowed for a CLI/ENV option to EXCLUDE specific cameras. In my case, I don't want to exclude a bunch: I want to only include a set (wildlife cams).

So: I added the option to **include** (`ONLY_CAMERAS`). Added everywhere incl. update of README.

I contemplated adding a remark about not combining EXCLUDE and ONLY but then I figured that it's quite odd to specify both, and left the warning out. 

Usage is pretty simple, like when excluding cameras:

```
 docker run --rm \
  -e UFP_USERNAME='yolo' \
  -e UFP_PASSWORD='yolo123' \
  -e UFP_ADDRESS='10.1.2.3' \
  -e UFP_SSL_VERIFY='false' \
  -e ONLY_CAMERAS='6724e2f9010eb703e4012964 643e94b501e67703e4002179 6724e84e02c4b703e4012c8d 6724ea16027cb703e4012d49 671e465e022c1403e40281f6' \
  -v '/tmp/unifi/data':'/data' \
  -v '/tmp/unifi/db':/config/database/ \
  ghcr.io/ep1cman/unifi-protect-backup
```

ep1cman commented:
Thanks for the pull request!

Some initial comments from a first quick glance:
- I do all the development on macOS so don't under why changes to the makefile are needed, works fine for me.
- if both exclude and only cameras are specified this should throw an error and refuse to start the application

WietseWind commented:
My pleasure, least I can do. I'll remove the additional docker makefile reference. Somehow on my machine (m3) docker always throws an error if multiple platforms are specified. I'll just run docker build outside of makefile then 👍 

I added the check for both args @ cli.py

When there's an argument error (more can be added in the future) the script exists with statuscode 200.

I modified the Docker (docker_root service file) config to **not** auto restart the script on exit code 200.

ep1cman commented:
Appologies its taken me so long to get around to properly review this, the holidays were super busy!

Im suprised that `click` does not include a way to construct mutually exlcusive groups of command line arguments and it had to be handled this way. Doing I quick google I found something like this: https://github.com/click-contrib/click-option-group. I think adding this would allow the code to remain cleaner.

Final comment, I am wondering if 'only_cameras' should just be "cameras" with the description saying that it defaults to all cameras if not specified.

Other than that looks good to me. The wildlife project is really cool!

WietseWind commented:
Agree at renaming to `cameras` instead of `only-cameras`. I tested `click option group`. It only works half, it does for args, but does not for environment variables. Tried to extend it with a custom type (`EnvAwareMutuallyExclusiveOptionGroup`) but I can't get it to behave.
